crates/networking/p2p/rlpx/frame.rs

```diff
@@ -28,28 +28,38 @@ pub(crate) async fn write<S: AsyncWrite + std::marker::Unpin>(
     header.resize(16, 0);
     state.egress_aes.apply_keystream(&mut header[..16]);
 
-    let header_mac_seed = {
-        let mac_digest: [u8; 16] = state.egress_mac.clone().finalize()[..16]
-            .try_into()
-            .unwrap();
-        let mut seed = mac_digest.into();
-        mac_aes_cipher.encrypt_block(&mut seed);
-        H128(seed.into()) ^ H128(header[..16].try_into().unwrap())
-    };
+    let header_mac_seed =
+        {
+            let mac_digest: [u8; 16] = state.egress_mac.clone().finalize()[..16]
```

Comment:
Maybe we should also use get here.

Reply:
I don't think it's necessary on egress_mac as they are type `Keccak256` and the `finalize()` implements `FixedOutput`.
I mean, we could fix it anyway, but there are a lot of places where we use them and the code would get unnecessarily cumbersome for checks that are redundant.